src/main/scala/uk/gov/hmrc/ct/ct600e/v2/E27.scala

```diff
@@ -21,11 +21,15 @@ import uk.gov.hmrc.ct.ct600e.v2.retriever.CT600EBoxRetriever
 
 case class E27(value: Option[Int]) extends CtBoxIdentifier("Value of any non-qualifying investments and loans") with CtOptionalInteger with Input with ValidatableBox[CT600EBoxRetriever] {
   override def validate(boxRetriever: CT600EBoxRetriever): Set[CtValidation] = {
```

Comment:
couldn't it just be something like:
validateAsMandatory ++ validateZeroOrPositiveInteger